modules/home/Hero.tsx

```diff
@@ -0,0 +1,39 @@
+import { ButtonLink } from '../atoms/button/Button';
+import { Edit } from '../icons/Edit';
+import { People } from '../icons/People';
+import styles from './Hero.module.css';
+
+export const Hero = () => (
+  <section className={styles.hero}>
+    <div className={styles.leftContainer}>
+      <h1>
+        <span>Meetup</span>
+        <br />
+        <strong>LyonJS</strong>
+      </h1>
+      <p>
+        La communauté lyonnaise
+        <br />
+        des utilisateurs de JavaScript
+      </p>
+
+      <div className={styles.buttons}>
+        <ButtonLink href="https://www.meetup.com/fr-FR/lyonjs/" target="_blank" rel="noreferrer noopener">
+          <People /> Rejoindre le meetup
+        </ButtonLink>
+        <ButtonLink variant="secondary" href="https://bit.ly/lyonjs-cfp" target="_blank" rel="noreferrer noopener">
+          Proposer un sujet <Edit />
+        </ButtonLink>
+      </div>
+    </div>
+
+    <img
+      src="/lion-1440.webp"
```

Comment:
are you ok for webp format ? [Safari support](https://caniuse.com/?search=webp) is 14+ on MacOS 11 Big Sur. It's like 4x lighter than png, so it would be cool to have it (since I compressed/cropped it manually on Squoosh, I did not bother to use `<source>` tags with multiple formats...)